mp/src/game/server/momentum/mom_chat_commands.cpp

```diff
@@ -0,0 +1,98 @@
+#include "cbase.h"
+
+#include "mom_chat_commands.h"
+#include "mom_player_shared.h"
+
+#include "tier0/memdbgon.h"
+
+const command_alias_t ChatCommands::aliases[] = {
```

Comment:
This should also be less hard-coded. Make a function that associates a chat command (and it's aliases) with a function.

An example of how this might look:
```cpp
RegisterChatCommand("restart", {"r"}, RestartCommand);

void RestartCommand(const CCommand& args)
{
    engine->ClientCommand(CMomentumPlayer::GetLocalPlayer()->edict(), "mom_restart 0");
}
```

Not sure if it's possible given our current toolchain, but you can also try to use lambdas.